manticore/platforms/evm.py

```diff
@@ -6,7 +6,7 @@
     SolverException
 from ..core.state import ForkState, TerminateState
 from ..utils.event import Eventful
-from ..core.smtlib.visitors import pretty_print, arithmetic_simplifier, translate_to_smtlib
+from ..core.smtlib.visitors import pretty_print, arithmetic_simplify, translate_to_smtlib
```

Comment:
Maybe we should use a simplifier with constant folding in the future. (so we remember) https://github.com/trailofbits/manticore/blob/dev-evm-arraystorage/manticore/core/smtlib/visitors.py#L483-L486